torch/csrc/Generator.cpp

```diff
@@ -230,6 +231,10 @@ static struct PyMemberDef THPGenerator_members[] = {
      nullptr},
     {nullptr}};
 
+static int THPGenerator_traverse(THPGenerator* self, visitproc visit, void* arg) {
+  return 0;
+}
```

Comment:
From my understanding the `THPGenerator` struct has no Python objects that we need to call `Py_VISIT` on?